source/_posts/analyzing-javascript-with-babel-traverse.md

```diff
@@ -0,0 +1,182 @@
+title: Detecting unused styles in JavaScript with `babel-traverse`
+author: Jared Forsyth
+date: 2017-04-08 15:47:36
+tags:
+---
+## Motivation
+
+Yesterday, a coworker asked what it would take to automatically detect and purge unused [aphrodite]() styles in our codebase.
```

Comment:
May also mention somewhere that since Aphrodite has ~the same API as RN's `StyleSheet` and other CSS-in-JS libraries, the same approach would generalize (this is sort of assumed on line 18, but might be helpful to clarify beforehand).